shared/chat/conversation/messages/react-button/emoji-picker/index.tsx

```diff
@@ -307,20 +302,27 @@ class EmojiPicker extends React.PureComponent<Props, State> {
     return sections ? (
       <>
         {this.getBookmarkBar(bookmarks)}
-        <Kb.SectionList
-          ref={this.sectionListRef}
-          desktopItemHeight={36}
-          desktopHeaderHeight={32}
-          keyboardShouldPersistTaps="handled"
-          initialNumToRender={14}
-          sections={sections}
-          desktopOnSectionChange={sectionIndex =>
-            this.mounted && this.setState({activeSectionIndex: sectionIndex})
-          }
-          stickySectionHeadersEnabled={Styles.isMobile}
-          renderItem={({item}: {item: Row; index: number}) => this.getEmojiRow(item, emojisPerLine)}
-          renderSectionHeader={({section}) => this.getSectionHeader(section.title)}
-        />
+        <Kb.Box2
+          key="section-list-container"
+          direction="vertical"
+          fullWidth={true}
+          style={styles.sectionListContainer}
+        >
```

Comment:
layout bug that hid the the last row under the footer because the bookmark pushed it down.